src/prelaunch/pieman.a

```diff
@@ -0,0 +1,33 @@
+;license:MIT
+;(c) 2019 by qkumba
+
+!cpu 6502
+!to "build/PRELAUNCH/PIEMAN",plain
+*=$106
+
+    !source "src/prelaunch/common.a"
+
+         lda   #$60       ; hook1
+         sta   $3EA9
+         jsr   $3E88      ; decompress
```

Comment:
Can we accelerate this?

Reply:
I'll enable and see if it works. Some games are a bit picky with the accelerator code.